client/src/core/client/admin/routes/Reports/ReportRowContainer.tsx

```diff
@@ -0,0 +1,99 @@
+import { useRouter } from "found";
+import React, { useCallback } from "react";
+import { graphql } from "react-relay";
+
+import { useDateTimeFormatter } from "coral-framework/hooks";
+import { withFragmentContainer } from "coral-framework/lib/relay";
+import {
+  NightMoonHalfIcon,
+  SignBadgeCircleIcon,
+  SvgIcon,
+} from "coral-ui/components/icons";
+import { RelativeTime, TableCell, TableRow } from "coral-ui/components/v2";
+
+import {
+  DSAReportStatus,
+  ReportRowContainer_dsaReport as DSAReportData,
+} from "coral-admin/__generated__/ReportRowContainer_dsaReport.graphql";
+
+interface Props {
+  dsaReport: DSAReportData;
+}
+
+const ReportRowContainer: React.FunctionComponent<Props> = ({ dsaReport }) => {
+  const formatter = useDateTimeFormatter({
+    day: "2-digit",
+    month: "2-digit",
+    year: "2-digit",
+    hour: "2-digit",
+    minute: "2-digit",
+  });
+  const { router } = useRouter();
+
+  const statusIconMapping = useCallback((status: DSAReportStatus | null) => {
+    if (!status) {
+      return null;
+    }
+    switch (status) {
+      case "AWAITING_REVIEW":
+        return <SvgIcon Icon={SignBadgeCircleIcon} />;
+      case "UNDER_REVIEW":
+        return <SvgIcon Icon={NightMoonHalfIcon} />;
+      case "COMPLETED":
```

Comment:
My only nitpick here would be that string constants are sometimes nicer when stuffed into a string type enum:

i.e.
```
export enum STATUS {
  AwaitingReview = "AWAITING_REVIEW",
  UnderReview = "UNDER_REVIEW",
  ...
}
```

Reply:
Updating to use `GQLDSAReportStatus` from our schema here, thanks for pointing out!